ports/geckolib/tools/regen_style_structs.sh

```diff
@@ -65,6 +65,7 @@ fi
   -no-type-renaming                                                 \
   -DMOZILLA_INTERNAL_API                                            \
   -DMOZ_STYLO_BINDINGS=1                                            \
+  -DJS_DEBUG=1                                                      \
```

Comment:
Well, we still need to deal with the fact that we're currently checking the output of the codegen into the tree. So if we just rely on mozilla-config.h, the output depends on whether the developer is running bindgen against a debug or release version of gecko.

I could we should just say that debug is the supported configuration right now, and solve this problem later.